docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/index.md

```diff
@@ -53,62 +53,61 @@ spec:
     spec:
       containers:
       - name: apache-container
-        image: httpd
+        image: httpd:2.4.35
 {{</ file >}}
 
-There are three main things to note in this ReplicaSet. The first is the `apiVersion`, which is `apps/v1`. This differs from the previous examples, which were all `apiVersion: v1`, because ReplicaSets do not exist in the `v1` core. They instead reside in the `apps` group of `v1`. The second and third things to note are the `replicas` field and the `selector` field. The `replicas` field defines how many replica Pods you want to be running at any given time. The `selector` field defines which Pods, matched by their label, will be controlled by the ReplicaSet.
-
-To view your ReplicaSets, issue the `get replicasets` command:
+The only noticeable difference between this Deployment and the example given in the [ReplicaSet](#replicasets) section is the `kind`. In this example we have chosen to initially install Apache 2.4.35. If you wanted to update that image to Apache 2.4.38, you would issue the following command:
```

Comment:
"As you will see, the only noticeable difference between a Deployment's manifest and that of a ReplicaSet is the `kind`." Just trying to point ahead to ReplicaSets and make it more specific.

Reply:
updated